content/docs/tasks/security/basic-access-control.md

```diff
@@ -25,26 +25,26 @@ This task shows how to control access to a service using the Kubernetes labels.
     Save the following YAML snippet as `route-rule-reviews-jason-v2-v3.yaml`:
 
     ```yaml
-    apiVersion: networking.istio.io/v1alpha3
-    kind: VirtualService
-    metadata:
-      name: reviews
-    spec:
-      hosts:
+      apiVersion: networking.istio.io/v1alpha3
```

Comment:
This entire yaml fragment should not be indented, it should start at column 0.

Reply:
@frankbu it's weird, the whole format will be messy when I delete the indent of yaml.

Reply:
I think this is a bug of hugo mardown parser, the inline code format will be messy when delete the indent. we can see similar issue in https://github.com/istio/istio.github.io/blob/master/content/help/troubleshooting.md#verify-prometheus-configuration